### ISSUE TITLE
Cannot run xacro via ros2 run in Windows

### ISSUE THREAD

**Issue by wmmc88** [ros/xacro#302]
```
ros2 run xacro xacro
Traceback (most recent call last):
  File "C:\dev\ros2_rolling\install\Scripts\ros2-script.py", line 33, in <module>
    sys.exit(load_entry_point('ros2cli==0.15.0', 'console_scripts', 'ros2')())
  File "C:\dev\ros2_rolling\install\Lib\site-packages\ros2cli\cli.py", line 89, in main
    rc = extension.main(parser=parser, args=args)
  File "C:\dev\ros2_rolling\install\Lib\site-packages\ros2run\command\run.py", line 70, in main
    return run_executable(path=path, argv=args.argv, prefix=prefix)
  File "C:\dev\ros2_rolling\install\Lib\site-packages\ros2run\api\__init__.py", line 64, in run_executable
    process = subprocess.Popen(cmd)
  File "c:\python38\lib\subprocess.py", line 858, in __init__
    self._execute_child(args, executable, preexec_fn, close_fds,
  File "c:\python38\lib\subprocess.py", line 1311, in _execute_child
    hp, ht, pid, tid = _winapi.CreateProcess(executable, args,
OSError: [WinError 193] %1 is not a valid Win32 application
```

this seems like the same error that happened with some other ros2 python programs: https://github.com/ros-visualization/rqt_console/issues/20

I think similar fixes there need to happen to xacro to be able to run this package under windows.

Directly invoking `xacro` also doesnt work since shebangs arent handled by windows.

**Comment by rhaschke** [comment]
Thanks for reporting the issue. As I don't have a Windows installation available, please apply the suggested changes yourself, validate, and file a PR.

**Comment by rhaschke** [comment]
Fixed via #304.
